recipe/conda_build_config.yaml

```diff
@@ -187,7 +187,7 @@ pin_run_as_build:
   geotiff:
     max_pin: x.x
   giflib:
-    max_pin: x.x
+    max_pin: x.x.x
```

Comment:
These 2 lines are unnecessary if you mark the build number=0 packages as broken

Reply:
You mean rely only on the `pin_subpackage` in the package and don't specify

```
giflib:
  max_pin: x.x.x
```

here? I'm OK with that.

Reply:
yes